docs/src/test/scala/docs/http/scaladsl/server/directives/JsonStreamingFullExamples.scala

```diff
@@ -4,8 +4,10 @@
 
 package docs.http.scaladsl.server.directives
 
+import com.github.ghik.silencer.silent
 import org.scalatest.wordspec.AnyWordSpec
 
+@silent("The outer reference in this type test cannot be checked at run time")
```

Comment:
I never ran into a bug because of this. Maybe we could just silence it globally as well?